main.nf

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env nextflow
+
+if(params.help) {
+    usage = file("$baseDir/USAGE")
+    cpu_count = Runtime.runtime.availableProcessors()
+
+    bindings = ["quick_registration":"$params.quick_registration",
+                "linear_registration":"$params.linear_registration",
+                "output_dir":"$params.output_dir",
+                "processes_register":"$params.processes_register",
+                "cpu_count":"$cpu_count"]
+
+    engine = new groovy.text.SimpleTemplateEngine()
+    template = engine.createTemplate(usage.text).make(bindings)
+
+    print template.toString()
+    return
+}
+
+log.info "Run Registration to template space"
+log.info "============================="
+log.info ""
+
+log.info ""
+log.info "Start time: $workflow.start"
+log.info ""
+
+log.debug "[Command-line]"
+log.debug "$workflow.commandLine"
+log.debug ""
+
+log.info "[Git Info]"
+log.info "$workflow.repository - $workflow.revision [$workflow.commitId]"
+log.info ""
+
+log.info "[Inputs]"
+log.info "Root: $params.input"
+log.info "Template: $params.template"
+log.info "Output directory: $params.output_dir"
+log.info ""
+
+log.info "Number of processes per tasks"
+log.info "============================="
+log.info "Template registration: $params.processes_register"
+log.info ""
+
+root = file(params.input)
+/* Watch out, files are ordered alphabetically in channel */
+Channel
+    .fromPath("$root/**/t1.nii.gz",
```

Comment:
There should be an `*` here before `t1.nii.gz` in order to deal properly with multiple subjects.